memcached.c

```diff
@@ -4191,6 +4287,837 @@ static inline void process_get_command(conn *c, token_t *tokens, size_t ntokens,
     }
 }
 
+// slow snprintf for debugging purposes.
+static void process_meta_command(conn *c, token_t *tokens, const size_t ntokens) {
+    assert(c != NULL);
+
+    if (tokens[KEY_TOKEN].length > KEY_MAX_LENGTH) {
+        out_string(c, "CLIENT_ERROR bad command line format");
+        return;
+    }
+
+    char *key = tokens[KEY_TOKEN].value;
+    size_t nkey = tokens[KEY_TOKEN].length;
+
+    item *it = limited_get(key, nkey, c, 0, false, DONT_UPDATE);
+    if (it) {
+        size_t total = 0;
+        size_t ret;
+        // similar to out_string().
+        memcpy(c->wbuf, "ME ", 3);
+        total += 3;
+        memcpy(c->wbuf + total, ITEM_key(it), it->nkey);
+        total += it->nkey;
+        c->wbuf[total] = ' ';
+        total++;
+
+        ret = snprintf(c->wbuf + total, c->wsize - (it->nkey + 12),
+                "exp=%d la=%llu cas=%llu fetch=%s cls=%u size=%lu\r\nEN\r\n",
+                (it->exptime == 0) ? -1 : (current_time - it->exptime),
+                (unsigned long long)(current_time - it->time),
```

Comment:
Looks like `la` is `seconds elapsed since the last access`, not the last access time.

Reply:
That's correct. It suffers from the same drift issue as exptime otherwise, so I changed it in the same way :(

Reply:
+1, update the protocol.txt?

Reply:
fixed